source/dev.jsx

```diff
@@ -17,7 +17,7 @@ const {
 } = getModule(location.pathname.replace(/\.html/, ''));
 
 // mock a server render
-document.querySelector('.root').innerHTML = Dom.renderToStaticMarkup(Component);
+Dom.render(Component, document.querySelector('.root'));
```

Comment:
This isn't necessarily a piece of your work, but you're touching it so I'm going to mention it:

`document.querySelector('html')` is in this file three times. Can we `const root = document.querySelector('html')` or something instead?

Reply:
Agree with Dan here.
Not really needed, but should we derive selecting body from this root const (instead of document) once added?
https://github.com/connectivedx/fuzzy-chainsaw/blob/4246ccdc74c44a9cdde1a2e21e0f60490e55746c/source/dev.jsx#L34